tools/misc/browse_dataset.py

```diff
@@ -55,6 +56,10 @@ def skip_pipeline_steps(config):
         ]
 
     cfg = Config.fromfile(config_path)
+
+    # update data root according to MMDET_DATASETS
+    update_data_root(cfg)
```

Comment:
Also need to modify analyze_results, benchmark, confusion_matrix, eval_metric, optimize_anchors and print_config